src/ray/util/util.cc

```diff
@@ -187,6 +187,9 @@ static std::vector<std::string> ParsePosixCommandLine(const std::string &s) {
 /// Python analog: None (would be shlex.split(s, posix=False), but it doesn't unquote)
 static std::vector<std::string> ParseWindowsCommandLine(const std::string &s) {
   RAY_CHECK(s.find('\0') >= s.size()) << "Invalid null character in command line";
+  if (s.empty()) {
+    return {};
+  }
```

Comment:
```suggestion
```
@fyrestone On a second thought, I think you probably want to remove this block. Instead, change the beginning of the loop from
```
  for (bool stop = false, in_dquotes = false; !stop;) {
```
to:
```
  while (i < j && (*i == ' ' || *i == '\t')) {
    ++i;  // Skip leading spaces
  }
  for (bool stop = i >= j, in_dquotes = false; !stop;) {
```
This is because Windows treats a leading space as 1 empty argument (and hence my implementation did the same). However, considering your goal, I think you want to return 0 arguments in that case as well.

In `ray/util/util_test.cc:59`, you will need to change the corresponding test case as well:
```
  ASSERT_EQ(ParseCommandLine(R"( a)", win32), ArgList({R"(a)"}));  // Differs from Windows
```